### PR TITLE
🌱 Change testdata suffix and regenerate samples

### PR DESCRIPTION
**Pull request by jason1028kr**
### Description
For #3110  Make corresponding changes in the `generate.sh` and fix samples.

### CODE REVIEW

File: test/testdata/generate.sh [hunk 1]
```diff
@@ -113,7 +113,7 @@ function scaffold_test_project {
       $kb create api --group example.com --version v1alpha1 --kind Busybox --image=busybox:1.28 --plugins="deploy-image/v1-alpha" --make=false
```
Comment (camilamacedo86) [inline test/testdata/generate.sh:113]: Hi @jason1028kr,

That is great! thank you for your contribution.

It is just missing one detail. Could you please also update the name in the tests: https://github.com/kubernetes-sigs/kubebuilder/blob/d887e25e0a62eae1fd5882267bc3a53535198c6d/test/testdata/test.sh#L47

**Review by camilamacedo86 — COMMENTED** [review]
Hi @jason1028kr,

That is great! thank you for your contribution.

It is just missing one detail. Could you please also update the name in the tests: https://github.com/kubernetes-sigs/kubebuilder/blob/d887e25e0a62eae1fd5882267bc3a53535198c6d/test/testdata/test.sh#L47

**Comment by jason1028kr** [comment]
ah, I've missed that. Changed test name and updated the emoji. Thank you @camilamacedo86

File: test/testdata/test.sh [hunk 2]
```diff
@@ -44,4 +44,4 @@ test_project project-v4-multigroup
 test_project project-v4-declarative-v1
 test_project project-v4-config
 test_project project-v4-with-deploy-image
-test_project project-v4-with-metrics
+test_project project-v4-with-grafana
```
Comment (camilamacedo86) [inline test/testdata/test.sh:47]: @jason1028kr :

Missing just updated here  `test_project project-v3-with-metrics` within `test_project project-v3-with-grafana`

**Comment by jason1028kr** [comment]
oh, didn't think it through that if block. Made the fix.

**Comment by camilamacedo86** [comment]
Thank you for your contribution 🥇.

It shows great for we move forward.  

/ok-to-test

/lgtm

/approved

**Review by camilamacedo86 — APPROVED** [review]

**Pull request merged**